src/assets/styles/tokens.scss

```diff
@@ -0,0 +1,36 @@
+:root {
+  --color-background-button-primary-default: var(--color-summer);
+  --color-background-button-primary-hover: var(--color-summer-dark-50);
+  --color-background-button-primary-focus: var(--color-summer);
+  --color-background-button-primary-hover-focus: var(--color-summer);
+  --color-background-button-primary-disabled: #cccccc;
+  --color-background-button-secondary-default: transparent;
+  --color-background-button-secondary-hover: var(--color-summer-light-20);
+  --color-background-button-secondary-focus: transparent;
+  --color-background-button-secondary-hover-focus: transparent;
+  --color-background-button-secondary-disabled: rgba(0, 0, 0, 0);
+  --color-border-button-primary-default: var(--color-summer);
+  --color-border-button-primary-hover: var(--color-summer-dark-50);
+  --color-border-button-primary-focus: var(--color-summer-dark-50);
+  --color-border-button-primary-hover-focus: var(--color-summer);
+  --color-border-button-primary-disabled: #cccccc;
+  --color-border-button-secondary-default: var(--color-summer);
+  --color-border-button-secondary-hover: var(--color-summer);
+  --color-border-button-secondary-focus: var(--color-summer);
+  --color-border-button-secondary-hover-focus: var(--color-summer);
+  --color-border-button-secondary-disabled: #808080;
+  --color-focus-outline-button-primary: #0072c6;
+  --color-focus-outline-button-secondary: #0072c6;
+  --color-label-button-primary-default: var(--color-black-90);
+  --color-label-button-primary-hover: var(--color-black-90);
+  --color-label-button-primary-focus: var(--color-black-90);
+  --color-label-button-primary-hover-focus: var(--color-black-90);
+  --color-label-button-primary-disabled: var(--color-black-90);
+  --color-label-button-secondary-default: var(--color-black-90);
+  --color-label-button-secondary-hover: var(--color-black-90);
+  --color-label-button-secondary-focus: var(--color-black-90);
+  --color-label-button-secondary-hover-focus: var(--color-black-90);
+  --color-label-button-secondary-disabled: #999898;
+  --color-background-checkbox-selected: var(--color-summer);
+  --color-background-checkbox-hover: var(--color-summer);
```

Comment:
No it will not work. I wonder what is the best approach to fix this for the future, is something like this ok: 

```      <Button
        variant="secondary"
        aria-label={t('child.form.modal.add.label')}
        className={styles.addChild}
        style={kukkuuStyles}
        iconLeft={<IconPlusCircle />}
        onClick={() => setIsOpen(true)}
      >
```

or do you have a better suggestion?

Reply:
Came up with a better solution.